fate/src/main/java/org/apache/accumulo/fate/AdminUtil.java

```diff
@@ -110,25 +110,36 @@ public String getTop() {
   public static class FateStatus {
 
     private final List<TransactionStatus> transactions;
-    private final Map<Long,List<String>> danglingHeldLocks;
-    private final Map<Long,List<String>> danglingWaitingLocks;
+    private final Map<String,List<String>> danglingHeldLocks;
+    private final Map<String,List<String>> danglingWaitingLocks;
+
+    private static Map<String,List<String>> convert(Map<Long,List<String>> danglocks) {
+      if (danglocks.isEmpty()) {
+        return Collections.emptyMap();
+      }
+
+      Map<String,List<String>> ret = new HashMap<>();
+      for (Entry<Long,List<String>> entry : danglocks.entrySet()) {
+        ret.put(String.format("%016x", entry.getKey()), Collections.unmodifiableList(entry.getValue()));
```

Comment:
> It's not clear, for example, that this is just pre-formatting the object for future logging.

Oh, this code is not used by logging.   This code is only used by test and the shell.  It uses the same formatting as the other code that does logging.